GEON-PPANG-iOS/Global/Utils/LayoutUtils.swift

```diff
@@ -49,4 +49,18 @@ final class LayoutUtils {
         
         return header
     }
+    
+    class func listConfiguration(appearance: UICollectionLayoutListConfiguration.Appearance,
+                                 headerMode: UICollectionLayoutListConfiguration.HeaderMode,
+                                 handler: UICollectionLayoutListConfiguration.ItemSeparatorHandler?
+    ) -> UICollectionLayoutListConfiguration {
```

Comment:
p4:
ㅎ 이렇게 띄울거면
```suggestion
    class func listConfiguration(
        appearance: UICollectionLayoutListConfiguration.Appearance,
        headerMode: UICollectionLayoutListConfiguration.HeaderMode,
        handler: UICollectionLayoutListConfiguration.ItemSeparatorHandler?
    ) -> UICollectionLayoutListConfiguration {
```
이렇게는 어떰 ?

Reply:
92bc8c1b0e2f731460707dc7c0131f4ab8604d34